examples/Multi_Armed_Bandit/bernoulli_multi_armed_bandit.py

```diff
@@ -0,0 +1,175 @@
+# - **Exploration:** On the other hand, you cannot know in advance how effective each option is until you have tested it.
+#
+# Bandit models offer a principled approach to handling this dilemma.
+# They can help you determine the effect size (e.g., the click-through rate) for your marketing options or optimize the effect size
+# while the campaign is still running. Both goals are achieved by dynamically selecting the version of the marketing campaign that is shown to each customer.
+
+### The Black Box Model
+
+# Abstractly, we can think of our problem as a black box system with a finite number of actions that can be triggered.
+# The image that comes to mind is that of a slot machine with multiple arms that can be pulled (hence the name "multi-armed bandit"), each offering a different win rate.
+# You can think of each arm as representing a version of the marketing campaign, and the corresponding win rates would be the true effect size of each marketing material version.
+
+# We can build such a black box system with just a few lines of code:
+
+
+@dataclass
+class MultiArmedBandit:
+    """A stateless black box system offering a finite number of actions (= arms)."""
+
+    win_rates: list[float]
+    """Win rates associated with the arms."""
+
+    def draw_arm(self, arm_index: int) -> bool:
+        """Draw a reward by pulling the specified arm."""
+        return bernoulli(self.win_rates[arm_index]).rvs()
+
+
+### Setup
+
+# For our example, we use a system with a fixed collection of win rates, which we
+# assume are unknown to us:
+
+WIN_RATES = [0.3, 0.4, 0.6, 0.65, 0.8]
+BANDIT = MultiArmedBandit(WIN_RATES)
+
+# Given this target system, we consider the following two tasks:
+# - **Online optimization:** Maximizing the total reward accumulated during a campaign
+# - **Active learning:** Estimating the win rates of all arms
+
+# Solving these task can be achieved through an appropriate choice of acquisition function:
+# For online optimization, we use {class}`~baybe.acquisition.acqfs.qThompsonSampling`.
+# For active learning, we switch to {class}`~baybe.acquisition.acqfs.PosteriorStandardDeviation`.
+
+# To estimate the corresponding effects, we simulate each campaign for a certain number of steps and repeat this process in multiple Monte Carlo runs:
+
+ACQFS = [
+    qThompsonSampling(),  # Online optimization
+    PosteriorStandardDeviation(),  # Active learning
+]
+N_MC_RUNS = 10
+N_ITERATIONS = 200
+
+
+### Building the Model
+
+# To model the rewards obtained from the bandit, we use a {class}`~baybe.targets.binary.BinaryTarget`.
+# The corresponding search space is spanned by a {class}`~baybe.parameters.categorical.CategoricalParameter` describing the bandit's arms.
+
+n_arms = len(WIN_RATES)
+objective = BinaryTarget(name="clicked").to_objective()
+searchspace = CategoricalParameter(
+    name="Bandit Arm", values=[f"arm_{i}" for i in range(n_arms)]
```

Comment:
this is the point where users hear/read about how ti sue this for the first time, any help they get and to avoid surprises helps
so unless `change later` is `next week` I would rate this helpful and would prefer removing it once expanded

Reply:
Yes, agree with Martin here.

Reply:
Ok, to me this is a bit like "schlafende Hunde wecken" but if you both insist. Have added it.